src/ResourceManager/Resources/Commands.Resources/Models.Authorization/AuthorizationClient.cs

```diff
@@ -183,10 +192,9 @@ public List<PSRoleDefinition> FilterRoleDefinitionsByCustom(string scope, bool s
             result.AddRange(AuthorizationManagementClient.RoleDefinitions.List(
                     scope,
 #if !NETSTANDARD
-                    parameters: new ListDefinitionFilterParameters { AtScopeAndBelow = scopeAndBelow })
-                .RoleDefinitions
+                    scopeAndBelow ? new ODataQuery<RoleDefinitionFilter>(filter => filter.AtScopeAndBelow()) : null)
```

Comment:
could you use the content that is already there i the preview branch first and make your changes on top of that
the if !NETSTANDARD and its else part is to differentiate the code compatibility and doesnt imply that both should be the same code